README.md

```diff
@@ -40,31 +40,33 @@ Your _**application.conf**_ should be configured to enable clamav scanning
 
 ```JavaScript
 clam.antivirus {            
-    enabled = true          
-    chunkSize = 32768       
-    host = avscan           
+    host = avscan
     port = 3310             
     timeout = 5000          
-    threadPoolSize = 20     
-    maxLength = 10485760    
 }
 ```
 
-Wire up your microservice to load the ClamAvConfig
+Wire up your microservice to load the ClamAvConfig using dependency injection within your Module. For an explanation on using Guice dependency injection within your Play project, see the [Play documentation](https://www.playframework.com/documentation/2.5.x/ScalaDependencyInjection).
 
 ```JavaScript
-object ClamAvConfiguration extends RunMode {
-
-  lazy val config: Option[Configuration]  = Play.current.configuration.getConfig(s"$env.clam.antivirus")
-  lazy val clamAvConfig = LoadClamAvConfig(config)
-
+class YourModule extends Module {
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] =
+    Seq(
+      ....
+      bind[ClamAvConfig].to[PlayClamAvConfig].eagerly(),
+      ....
+    )
 }
 ```
 
 Use ClamAntiVirus
 
 ```JavaScript
-def clamAv: ClamAntiVirus = new ClamAntiVirus()(ClamAvConfiguration.clamAvConfig)
-clamAv.send(stream)
-clamAv.checkForVirus()
+import javax.inject.Inject
+import uk.gov.hmrc.clamav.ClamAntiVirus
+
+class YourClass @Inject()(clamAv: ClamAntiVirus) {
+  clamAv.send(stream)
```

Comment:
It's about existing API but afaik send(stream) and checkForVirus() might be one function.
Moreover send() opens the TCP connection while checkForVirus() terminates it, so if we fail to do that, we might run out of resources.

Reply:
Yes, this is weird. I can't work out when one would use "send" instead of "checkForVirus". Odd...